microsoft/testsuites/libvirt/libvirt_tck.py

```diff
@@ -0,0 +1,46 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from pathlib import Path
+
+from lisa import (
+    Environment,
+    Logger,
+    Node,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+)
+from lisa.testsuite import TestResult
+from lisa.tools import Lscpu
+from lisa.util import SkippedException
+from microsoft.testsuites.libvirt.libvirt_tck_tool import LibvirtTck
+
+
+@TestSuiteMetadata(
+    area="libvirt",
+    category="community",
+    description="""
+    Runs the Libvirt TCK (Technology Compatibility Kit) tests.
+    """,
+)
+class LibvirtTckSuite(TestSuite):
+    @TestCaseMetadata(
+        description="""
+            Runs the Libvirt TCK (Technology Compatibility Kit) tests.
+        """,
+        priority=3,
+    )
+    def verify_libvirt_tck(
```

Comment:
add NestedVirtualization feature in requirement, so that, case can be run with Virtualization capability.

Reply:
I am already doing the check in the case and skipping if virtualization is not enabled.

Reply:
Adding it in requirements causes the test case to be skipped for ready platform.